datasets/gov/lsd/pasiekiamumo_teritorijos.csv

```diff
@@ -22,8 +21,7 @@ Sudarant 15 min pasiekiamumo automobiliu teritorijas, neatsižvelgta į piko val
 ,,,,,,enum,,,"""pėsčiomis""",,,,,
 ,,,,,,,,,"""automobiliu""",,,,,
 ,,,,,naujinimo_data,date,D,,,4,open,,Duomenų naujinimo data,
-,,,,,geometrija,geometry,,,,2,open,,Objekto geometrija WKT formatu (LKS94),
-,,,,,,comment,type,,"update(type: ""geometry(polygon, 3346)"")",3,open,vadovas:dsa/duomenu-tipai.html#erdviniai-duomenys,,
+,,,,,geometrija,"geometry(polygon, 3346)",m,,,4,open,,Objekto geometrija WKT formatu (LKS94),

```

Comment:
Negalima keisti į `polygon`, kadangi duomenų bazėje yra `polygon` ir `multipolygon`.

```
 st_geometrytype | count 
-----------------+-------
 ST_MultiPolygon |    17
 ST_Polygon      |  2131
```

Vietoj `polygon` turėtu būti `geometry`.

Be to, testuodamas pastebėjau, kad koordinatės veda į Indijos vandenyną. Ta reikėtu ištestuoti ir nustatyti teisingą SRID ir x, y eiliškumą.

Pabandžiau sukeisti koordinates vietomis, bet vis tiek klaidingai rodo:

https://www.openstreetmap.org/?mlat=3.058898265434481&mlon=68.84969634817101#map=19/3.05890/68.84970
https://www.openstreetmap.org/?mlat=68.84969634817101&mlon=3.058898265434481#map=19/68.84970/3.05890

Matyt bus SRID problema.

Reply:
Gal reikia padaryti konversiją iš 4326 į 3346 SRID?

Reply:
Pakeista į `geometry`, SRID atitinka LKS94, bet panašu, kad netinkamas eiliškumas, pabandysiu pakeisti duomenyse